src/app/productConfig/productConfig.modal.js

```diff
@@ -19,7 +19,7 @@ class ModalInstanceCtrl{
   }
 
   frequencyOrder(f){
-    let order = ['NA', 'MON', 'QUARTERLY', 'ANNUAL', 'SEMIANNUAL'];
+    let order = ['NA', 'MON', 'QUARTERLY', 'SEMIANNUAL', 'ANNUAL'];
```

Comment:
Do what you think is best. It just seemed a little weird sorting by it and then never showing it.